private_gpt/components/vector_store/vector_store_component.py

```diff
@@ -36,22 +40,58 @@ class VectorStoreComponent:
     vector_store: VectorStore
 
     @inject
-    def __init__(self) -> None:
-        chroma_settings = Settings(anonymized_telemetry=False)
-        chroma_client = chromadb.PersistentClient(
-            path=str((local_data_path / "chroma_db").absolute()),
-            settings=chroma_settings,
-        )
-        chroma_collection = chroma_client.get_or_create_collection(
-            "make_this_parameterizable_per_api_call"
-        )  # TODO
+    def __init__(self, settings: Settings) -> None:
+        match settings.vectorstore.database:
+            case "chroma":
+                chroma_settings = ChromaSettings(anonymized_telemetry=False)
+                chroma_client = chromadb.PersistentClient(
+                    path=str((local_data_path / "chroma_db").absolute()),
+                    settings=chroma_settings,
+                )
+                chroma_collection = chroma_client.get_or_create_collection(
+                    "make_this_parameterizable_per_api_call"
+                )  # TODO
 
-        self.vector_store = typing.cast(
-            VectorStore,
-            BatchedChromaVectorStore(
-                chroma_client=chroma_client, chroma_collection=chroma_collection
-            ),
-        )
+                self.vector_store = typing.cast(
+                    VectorStore,
+                    BatchedChromaVectorStore(
+                        chroma_client=chroma_client, chroma_collection=chroma_collection
+                    ),
+                )
+
+            case "qdrant":
+                try:
+                    from llama_index.vector_stores.qdrant import QdrantVectorStore
+                    from qdrant_client import QdrantClient
```

Comment:
Given the installation of the package is optional, it won't be installed by the CI. 
The best solution I can think of is to make mypy ignore that import. 

```suggestion
                    from qdrant_client import QdrantClient  # type: ignore
```